### ISSUE TITLE
low  accuracy

### ISSUE THREAD

**Issue by beihaidao** [yjxiong/temporal-segment-networks#112]
I run: "bash scripts/train_tsn.sh ucf101 flow "  to train ,the N_GPU=1 & bach_size=8.the training result is:
I0706 23:48:09.203856  3579 solver.cpp:519] Snapshotting solver state to models/ucf101_split1_tsn_flow_bn_inception_iter_18000.solverstate
I0706 23:48:09.566228  3579 solver.cpp:415] Iteration 18000, loss = 4.04424
I0706 23:48:09.566251  3579 solver.cpp:433] Iteration 18000, Testing net (#0)
I0706 23:48:47.075711  3579 solver.cpp:490]     Test net output #0: accuracy = 0.109474
I0706 23:48:47.075783  3579 solver.cpp:490]     Test net output #1: loss = 3.80083 (* 1 = 3.80083 loss)
I0706 23:48:47.075788  3579 solver.cpp:420] Optimization Done.
I0706 23:48:47.075790  3579 caffe.cpp:203] Optimization Done.
Setting device 0
from this logs ,the accuracy=0.109474

**Comment by yjxiong** [comment]
In your setting, you have an effective batchsize of 8, while the original setting has 128. This means you trained the models with much fewer epochs than needed. Try using more GPUs and a larger batchsize to have an effective batchsize of 128. `iter_size` can also help to compensate if you are limited by hardware resources.
